pkg/solana/config/config.go

```diff
@@ -31,6 +31,7 @@ var defaultConfigSet = Chain{
 	BlockHistoryPollPeriod:   config.MustNewDuration(5 * time.Second),
 	ComputeUnitLimitDefault:  ptr(uint32(200_000)), // set to 0 to disable adding compute unit limit
 	EstimateComputeUnitLimit: ptr(false),           // set to false to disable compute unit limit estimation
+	BlockHistoryDepth:        ptr(uint64(20)),      // number of blocks to fetch for multiple blocks fee estimation
```

Comment:
Just to keep things consistent with EVM where we can, could we rename this config to `BlockHistorySize` instead?

Reply:
Also to keep existing behavior of the BHE, we should maybe set this default to 1 to mimic latest block behavior.